tests/summary_eff_fac.DATA

```diff
@@ -0,0 +1,138 @@
+START
+10 MAI 2007 /
+RUNSPEC
+
+TITLE
+SUMMARYTESTS
+
+-- A simple 10x10x10 cube. Simple to reason about, large enough for all tests
+DIMENS
+ 10 10 10 /
+
+REGDIMS
+  10 /
+
+OIL
+GAS
+WATER
+
+GRID
+
+DX
+1000*1 /
+DY
+1000*1 /
+DZ
+1000*1 /
+TOPS
+100*1 /
+
+
+PORO
+1000*0.2 /
+
+REGIONS
+
+FIPNUM
+ 1*2
+/
+
+SUMMARY
+DATE
+
+ROPR
+/
+
+RWIR
+/
+
+WOPR
+/
+WOPT
+/
+WOIR
+/
+WOIT
+/
+
+GOPR
+/
+GOPT
+/
+GOIR
+/
+GOIT
+/
+GWCT
+/
+GGOR
+/
+
+FOPR
+
+FOPT
+
+FOIR
+
+FOIT
+
+COPR
+'W_2' /
+/
+
+COPT
+'W_2' /
+/
+
+SCHEDULE
+
+GRUPTREE
+  'G_1'  'G' /
+  'G_2'  'G' /
+  'G_3'  'G_4' /
+/
+
+
+WELSPECS
+     'W_1'        'G_1'   1    1  3.33       'OIL'  7* /
+     'W_2'        'G_2'   2    1  3.33       'OIL'  7* /
+     'W_3'        'G_3'   2    1  3.33       'OIL'  7* /
+/
+
+WEFAC
```

Comment:
It looks like you are missing a test of the simplest case - a well with WEFAC and no GEFAC?